test/test_torch.py

```diff
@@ -13395,6 +13395,25 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    @onlyCUDA
```

Comment:
should probably run on all devices -- there's nothing specific about CUDA here (just that the bug happened to be in CUDA).

Reply:
We should rename the test to reference "index_add" if we remove the @onlycuda, the TH code doesn't use "atomic add" (it does the computation serially)